### DISCUSSION TITLE
5.11.1

### DISCUSSION THREAD

**Discussion by arturcic** [GitTools/GitVersion#3276] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/GitTools/GitVersion/releases/tag/5.11.1'>5.11.1</a>.</em>

**Comment by JSaaS** [comment]
We got into a bit of problem with the latest versions: 

 INFO [11/08/22 12:38:16:67] Working directory: /home/vsts/work/1/s
INFO [11/08/22 12:38:16:70] Project root is: /home/vsts/work/1/s/
INFO [11/08/22 12:38:16:70] DotGit directory is: /home/vsts/work/1/s/.git
INFO [11/08/22 12:38:16:70] Branch from build environment: refs/heads/features/12708
INFO [11/08/22 12:38:16:70] Begin: Normalizing git directory for branch 'refs/heads/features/12708'
  INFO [11/08/22 12:38:16:73] One remote found (origin -> 'https://<REMOVED FOR PRIVACY REASONS>').
  INFO [11/08/22 12:38:16:74] Skipping fetching, if GitVersion does not calculate your version as expected you might need to allow fetching or use dynamic repositories
  INFO [11/08/22 12:38:16:74] Creating local branch features/12708
  INFO [11/08/22 12:38:16:77] Creating local branch from remote tracking 'refs/remotes/origin/643014468273925a5b620549c43694448f58579b'.
  INFO [11/08/22 12:38:16:78] HEAD points at branch 'refs/heads/features/12708'.
  INFO [11/08/22 12:38:16:78] End: Normalizing git directory for branch 'refs/heads/features/12708' (Took: 78.90ms)
  INFO [11/08/22 12:38:16:79] Begin: Loading version variables from disk cache
    INFO [11/08/22 12:38:16:79] Cache file /home/vsts/work/1/s/.git/gitversion_cache/E30BF4A54921B6B95A9898D67B0D31CAF3430D1F.yml not found.
  INFO [11/08/22 12:38:16:79] End: Loading version variables from disk cache (Took: 0.36ms)
  INFO [11/08/22 12:38:16:79] Using latest commit on specified branch
  INFO [11/08/22 12:38:16:90] Running against branch: features/12708 (6430144 <REMOVED COMMENT FOR PRIVACY REASONS>)
  INFO [11/08/22 12:38:16:91] Begin: Calculating base versions
  INFO [11/08/22 12:38:16:92] End: Calculating base versions (Took: 8.25ms)
  WARN [11/08/22 12:38:16:92] An error occurred:
**Failed to parse 1.0 into a Semantic Version**

Reverting to using 5.10.3 makes it work again just as it should. Any ideas?

**Reply by asbjornu** [reply]
Are you certain that you are using version 5.11.1, @JSaaS?

**Reply by JSaaS** [reply]
Not 100% actually. Our deployment script states:

# GitVersion Setup
            - task: gitversion/setup@0
              displayName: Git - Version - Init
              inputs:
                versionSpec: '5.x'

But, it did start making that error just about the same time as 5.11.1 got released.

**Reply by asbjornu** [reply]
We have reports in #3273 that `5.11.1` fixed this so please ensure that you're using that version, @JSaaS.
